easy_my_coop_loan/models/loan.py

```diff
@@ -183,7 +172,7 @@ def compute_loan_interest(self):
                 }
             self.get_interest_vals(line, vals)
 
-            rounded_term = int(self.term.term)
-            if self.term.term - rounded_term > 0:
+            rounded_term = int(self.loan_term)

```

Comment:
if you're rounding here and at line 140, maybe you should use an integer instead of a float?

Reply:
Nope I'm rounding it just to know if the term is of type integer and to handle it like this. In the case of the term is 4,5 years we should take that case too. It's planned but not yet implemented. So for me it should stay of type float for now.